### PR TITLE
BACK-1209: Add 'logs' and 'hosts' commands

### PR DESCRIPTION
**Pull request by sjl2024**
The 'hosts' commands fetches any docker host IDs that correspond to internal DLC logs. The 'logs' command is a fully-interactive command which prompts users for query information - DLC host ID, log start datestamp (optional), log end datestamp (optional) - and displays resulting logs to the user (along with a total count at the end)

### CODE REVIEW

**Comment by mjsalinger** [comment]
Can we remove the `hosts` command?

**Comment by sjl2024** [comment]
Yes sir. Per our recent Slack discussion with Ivan, I'm in the process of removing the `hosts` command, as well as the `logs` prompt which allows a user to select a host (to only retrieve log entries for that container).

File: README.md [hunk 1]
```diff
@@ -11,8 +11,10 @@ In your project directory, run `kinvey-dlc-cli config` to set-up your project. T
 * `config` - set project options.
 * `deploy` - deploy the current project as a Kinvey-backed Data Link Connector. You can check the status of a deploy using the `status` command.
 * `help` - display usage information.
+* `hosts` - display list of hosts that this DLC has executed on
```
Comment (mjsalinger) [inline README.md]: Still a reference to this command

File: lib/datalink.coffee [hunk 2]
```diff
@@ -117,6 +147,31 @@ class Datalink
...
+# Executes a GET /apps/:app/datalinks/:datalink/logs request.
+  _execDatalinkLogs: (cb) =>
+    paramAdded = false
+    url = "/v#{project.schemaVersion}/data-links/#{project.datalink}/logs"
+
+    logger.debug "Log start timestamp: #{this.logStartTimestamp}"
+    logger.debug "Logs end timestamp: #{this.logEndTimestamp}"
+
+    # Request URL creation (versus user input params)
+    if !!this.logStartTimestamp
```
Comment (mjsalinger) [inline lib/datalink.coffee]: Double-negative?

**Comment by mjsalinger** [comment]
LGTM

**Pull request merged**